spec/notifiers/user_job_match_notifier_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe UserJobMatchNotifier, type: :mailer do
+  let(:mailer) { Struct.new(:deliver_later).new(nil) }
+
+  it  'must work' do
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.